cypress/integration/TextArea/Can_set_a_value/index.js

```diff
@@ -0,0 +1,19 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a TextArea with no text is rendered', () => {
+    cy.visitStory('TextArea', 'Default')
+    cy.verifyFormValue('comment', undefined)
+})
+
+When('the user types something', () => {
+    cy.wrap('user input').as('userInput')
+    cy.get('@userInput').then(userInput => {
```

Comment:
Same as above for passing around @userInput. It doesn't seem like you gain that much over just using `'user input'` directly.